dist/bitburner.d.ts

```diff
@@ -6187,12 +6207,12 @@ export declare interface WarehouseAPI {
      * @param all - Sell in all city
      */
     sellProduct(
-    divisionName: string,
-    cityName: string,
-    productName: string,
-    amt: string,
-    price: string,
-    all: boolean,
+        divisionName: string,
```

Comment:
Looks like someone did not format?  Is it okay for me to have formatted the file?